### ISSUE TITLE
Possible Issue in PinOut.SVG 

### ISSUE THREAD

**Issue by xenon2008** [FYSETC/FYSETC-SPIDER#39]
Good Morning,

I am planing to use one of the RGB Ports for a Chamber Fan on my Voron 2.4 which i am building currently.
Because i already had all 3 FAN Ports in use, i think over to use a RGB Port for a additional Fan (Chamber Fan)

I noticed that in the [PinOut.SVG ](https://raw.githubusercontent.com/FYSETC/FYSETC-SPIDER/main/Spider_V1.0_Pinout.svg) the Negative Pin is marked as Red? 
![grafik](https://user-images.githubusercontent.com/12835450/128625664-e6e59603-58be-4923-8099-2f7eb0c0d3b9.png)

And in the [Wiring.SVG](https://raw.githubusercontent.com/FYSETC/FYSETC-SPIDER/main/Spider_v1.0_wiring.svg) i think that "+" & "-" are reversed?
![grafik](https://user-images.githubusercontent.com/12835450/128625717-ba3a8d27-79d6-4da1-bcf4-b194d0e23e2d.png)

I am not 100% sure, but if i understand this right "+" should be left and "-" right? - regarding to this Diagram here:
![grafik](https://user-images.githubusercontent.com/12835450/128625745-58900e0f-bafb-4f3b-b2d8-94c687441caa.png)

**Comment by bigsubgit** [comment]
Hello, I'm starting now to studying the spider for my new 2.4, how could be a negative pole if there we put a jumper cap?

**Comment by bigsubgit** [comment]
Sorry, perhaps I was wrong in my previous post.

**Comment by GerogeFu** [comment]
All '+' is 12v/24v , and '-' is GND

**Comment by xenon2008** [comment]
> 
> 
> All '+' is 12v/24v , and '-' is GND

Sure, but i mean the color on the first picture.... GND is RED?
and on the second Picture i think the in the Green & Blue Section "+" & "-" are reversed?

**Comment by GerogeFu** [comment]
No, the middle point is not GND, it is common point for jumper, so if you can close a jumper on 24v and the middle point or on 12v and the middle point. That means you can select voltage.

**Comment by GerogeFu** [comment]
> Hello, I'm starting now to studying the spider for my new 2.4, how could be a negative pole if there we put a jumper cap?

jumper cap

**Comment by xenon2008** [comment]
> 
> 
> No, the middle point is not GND, it is common point for jumper, so if you can close a jumper on 24v and the middle point or on 12v and the middle point. That means you can select voltage.

oh sorry my Mistake!
But it is also right that on the two Blue Ports  the Voltage Pin is right?

**Comment by GerogeFu** [comment]
Stick to this: all '+' is 12v/24v, and all '-' is GND. Then you won't make mistake.
